web/modules/custom/joinup_migrate/joinup_migrate.post_update.php

```diff
@@ -159,3 +161,66 @@ function joinup_migrate_post_update_more_redirects() {
     }
   }
 }
+
+/**
+ * Switch the ownership in 'Sharing and reuse of IT solutions' [ISAICP-3980].
+ */
+function joinup_migrate_post_update_srits_change_owner() {
+  $collection_id = 'http://data.europa.eu/w21/e5f0febe-f29c-428a-a1a2-6bb0ccd37949';
+  // The 'Joinup moderator'.
+  $current_owner_id = 700003;
+  // The 'Joinup editor'.
+  $new_owner_id = 6363;
+  /** @var \Drupal\og\MembershipManagerInterface $membership_manager */
+  $membership_manager = \Drupal::service('og.membership_manager');
+  /** @var \Drupal\Core\Entity\EntityTypeManager $entity_type_manager */
+  $entity_type_manager = \Drupal::service('entity_type.manager');
+
+  /** @var \Drupal\rdf_entity\RdfInterface $collection */
+  $collection = $entity_type_manager->getStorage('rdf_entity')->load($collection_id);
```

Comment:
Could you add a condition to not run the update if $collection === NULL? Because this breaks the update process on UAT, where the this collection doesn't exist.